browserstack.browsers.js

```diff
@@ -1,4 +1,15 @@
 const browserstackBrowsers = {
+  // iOS
+  iphone: {
```

Comment:
Happy for this to be the fix, though. Let's get this in and bump Skate. When Skate's green, let's push a 1.0!